jwst/regtest/conftest.py

```diff
@@ -24,12 +24,22 @@
 @pytest.fixture(scope="session")
 def artifactory_repos(pytestconfig):
     """Provides Artifactory inputs_root and results_root"""
-    try:
-        inputs_root = pytestconfig.getini('inputs_root')[0]
-        results_root = pytestconfig.getini('results_root')[0]
-    except IndexError:
+    inputs_root = pytestconfig.getini('inputs_root')
+    # in pytest 8 inputs_root will be None
+    # in pytest <8 inputs_root will be []
+    # see: https://github.com/pytest-dev/pytest/pull/11594
+    # using "not inputs_root" will handle both cases
+    if not inputs_root:
```

Comment:
following up on an offline conversation with @nden

I added some comments about the change here. Let me know if it would be clearer to change this conditional to compare explicitly against `None` and `[]`. I'm happy to make the change.